waku/v2/protocol/relay/waku_relay_test.go

```diff
@@ -56,6 +60,83 @@ func TestWakuRelay(t *testing.T) {
 	<-ctx.Done()
 }
 
+func createRelayNode(t *testing.T) (host.Host, *WakuRelay) {
+	port, err := tests.FindFreePort(t, "", 5)
+	require.NoError(t, err)
+	host, err := tests.MakeHost(context.Background(), port, rand.Reader)
+	require.NoError(t, err)
+
+	relay := NewWakuRelay(nil, 0, timesource.NewDefaultClock(), utils.Logger())
+	relay.SetHost(host)
+	return host, relay
+}
+
+func TestGossipsubScore(t *testing.T) {
+	testTopic := "/waku/2/go/relay/test"
+
+	hosts := make([]host.Host, 5)
+	relay := make([]*WakuRelay, 5)
+	for i := 0; i < 5; i++ {
+		hosts[i], relay[i] = createRelayNode(t)
+		if i == 0 {
+			// This is a hack to remove the default validator from the list of default options
+			relay[i].opts = relay[i].opts[:len(relay[i].opts)-1]
```

Comment:
Because the default validator works also against my own messages, so with it active I wouldnt be able to send random payloads in this test unit (since the default validator expects `WakuMessage`s). With some refactoring I probably could support this, but I did not like much the idea of spending much time adding extra parameters or support functions to `WakuRelay` to allow disabling the default validator in test units.
In the future if we require to add more options to `WakuRelay` it might be a good moment to revisit this decision.